src/_base/helm/app/templates/service/varnish/statefulset.yaml

```diff
@@ -15,6 +15,8 @@ spec:
   template:
     metadata:
       creationTimestamp: null
+      annotations:
+        checksum/config: {{ include (print $.Template.BasePath "/service/varnish/configmap.yaml") . | sha256sum }}
```

Comment:
some have experienced the sha256 is too long for an annotation, but hopefully that is just in a version lower than EKS

Reply:
Could do `sha1sum`?

Reply:
seems to have changed 4 years ago to 256 characters. sha256 is 64 (where limit seemed to trigger as if annotation was 65)

Reply:
k8s 1.1 was the version that was released in, so I think we're safe